gensim/parsing/preprocessing.py

```diff
@@ -125,17 +317,84 @@ def stem_text(text):
 
 
 def preprocess_string(s, filters=DEFAULT_FILTERS):
+    """Takes string, applies list of chosen filters to it, where filters are methods from this module. Default list of filters consists of: strip_tags, strip_punctuation, strip_multiple_whitespaces, strip_numeric, remove_stopwords, strip_short, stem_text. <function <lambda>> in signature means that we use lambda function for applying methods to filters.
```

Comment:
Use references to this, not raw text, i.e. 
```
:func:`~gensim.parsing.preprocessing.strip_tags`
```
instead of `strip_tags` (here and anywhere)